demos/rag_agentic/notebooks/Level4_agentic_and_mcp.ipynb

```diff
@@ -580,18 +630,23 @@
     "# Create simple agent with tools\n",
     "agent = Agent(\n",
     "    client,\n",
-    "    model=granite_model, # replace this with your choice of model\n",
-    "    instructions = sys_prompt3 , # update system prompt based on the model you are using\n",
-    "    tools=[\"mcp::openshift\"],\n",
+    "    model=llama_model, # replace this with your choice of model\n",
+    "    instructions = sys_prompt4,\n",
+    "    tools=[\"mcp::openshift\", \"mcp::slack\"],\n",
     "    tool_config={\"tool_choice\":\"auto\"},\n",
-    "    sampling_params={\"max_tokens\":4096}\n",
+    "    sampling_params={\n",
+    "        \"max_tokens\":4096,\n",
+    "        \"strategy\": {\"type\": \"greedy\"},\n",
+    "    }\n",
     ")\n",
     "\n",
-    "user_prompts = [\"View the logs for pod 'hello-world' in the llama-serve namespace. Categorize it as 'Normal' or 'Error'. You must strictly answer with the pod name and category normal or error in a table format and also briefly explain why you categorized it as normal or error.\",\n",
-    "               \"View the logs for pod 'test-pod' in the llama-serve namespace. Categorize it as 'Normal' or 'Error'. You must strictly answer with the pod name and category normal or error in a table format and also briefly explain why you categorized it as normal or error.\"]\n",
+    "user_prompts = [\"View the logs for pod hello-world in the llama-serve OpenShift namespace. Categorize it as normal or error.\",\n",
+    "               \"Summarize the results with the pod name, category along with a briefly explaination as to why you categorized it as normal or error. Respond with plain text only. Do not wrap your response in additional quotation marks.\",\n",
+    "               \"Send a message with the summarization to the demos channel on Slack.\"]\n",
+    "\n",
     "session_id = agent.create_session(session_name=\"OCP_demo\")\n",
     "\n",
-    "for prompt in user_prompts:\n",
+    "for i, prompt in enumerate(user_prompts):\n",
```

Comment:
What is the purpose of this change? I don't see where `i` is used in this for-loop?

Reply:
yeah I actually I needed it for just printing the output for every turn (omitted the print statement here), but maybe I'll just remove `i`